### PR TITLE
View models for admin#auctions_controller

### PR DESCRIPTION
**Pull request by jessieay**
* I can see the light at the end of the tunnel

### CODE REVIEW

File: app/controllers/admin/auctions_controller.rb [hunk 1]
```diff
@@ -2,23 +2,25 @@ class Admin::AuctionsController < ApplicationController
...
   def index
-    @auctions = Auction.all.map { |auction| AdminAuctionPresenter.new(auction) }
+    auctions = Auction.all
+    @auctions = auctions.map { |auction| Admin::AuctionListItem.new(auction) }
 
     respond_to do |format|
       format.html
       format.json do
-        render json: @auctions, each_serializer: Admin::AuctionSerializer
+        render json: auctions, each_serializer: Admin::AuctionSerializer
```
Comment (harrisj) [inline app/controllers/admin/auctions_controller.rb]: I think we need to consider separate controllers for the API

Comment (jessieay) [inline app/controllers/admin/auctions_controller.rb]: absolutely on board with that plan!

Comment (harrisj) [inline app/controllers/admin/auctions_controller.rb]: Sorry, @adelevie, I know this is rough

File: spec/controllers/admin/auctions_controller_spec.rb [hunk 2]
```diff
@@ -10,8 +10,7 @@
       auctions = assigns(:auctions)
       expect(auctions).to be_a(Array)
       auction = auctions.first
-      expect(auction).to be_a(AdminAuctionPresenter)
-      expect(auction.id).to eq(auction_record.id)
+      expect(auction.auction).to eq(auction_record)
```
Comment (harrisj) [inline spec/controllers/admin/auctions_controller_spec.rb]: Our old friend `auction.auction` is back, but I suppose if it's a test I'm fine

Comment (harrisj) [inline spec/controllers/admin/auctions_controller_spec.rb]: Do we care about checking the type of the object or is that not really useful?

**Comment by harrisj** [comment]
Going to merge now, but it'll mean having to rebase my own datepicker PR

**Pull request merged**